changelog.md

```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+# [4.3.0] - 2020-07-28
+
+### Added
+- Added new d.ts bundler rollup plugin, that wraps the `rollup-plugin-dts` plugin and does additional cleanup and validation.
+
+### Changed
+- Updated the minimum rollup version to 2.0.0.
```

Comment:
Is that a minor change?

Reply:
Yes, as it doesn't impact the public API.

Reply:
What if someone is using swag and rollup 1 — then a minor upgrade could break their build

Reply:
It wouldn't as that's not how node works... it'll bundle it and resolve it specifically for swag in that case.

Reply:
Also, remember swag is not compatible with rollup as it's meant to provide the interface to the rollup version it's using.